proarc-common/src/main/java/cz/cas/lib/proarc/common/export/mets/MetsUtils.java

```diff
@@ -172,18 +169,20 @@ private static void findChildPSPs(DigitalObject dObj, MetsContext ctx, List<Stri
     }
 
     public static String getElementType(String model) {
-     String type = Const.typeMap.get(model);
-     if (type == null) {
-         List<NdkEbornPlugin> plugins = MetaModelRepository.getInstance().find().stream().map(metaModel -> metaModel.getPlugin()).distinct()
-                 .filter(plugin -> plugin instanceof NdkEbornPlugin).map(plugin -> ((NdkEbornPlugin) plugin)).collect(Collectors.toList());
-         for (NdkEbornPlugin plugin : plugins) {
-            if (plugin.TYPE_MAP.containsKey(model)) {
-                return plugin.TYPE_MAP.get(model);
+        String type = Const.typeMap.get(model);
+        if (type == null) {
+            List<NdkEbornPlugin> plugins = MetaModelRepository.getInstance().find().stream().map(metaModel -> metaModel.getPlugin()).distinct()
+                    .filter(plugin -> plugin instanceof NdkEbornPlugin).map(plugin -> ((NdkEbornPlugin) plugin)).collect(Collectors.toList());
+            for (NdkEbornPlugin plugin : plugins) {
+                if (plugin.TYPE_MAP.containsKey(model)) {
+                    return plugin.TYPE_MAP.get(model);
+                }
             }
-         }
-     }
-     return type;
+            throw new NoSuchElementException("unknown element type for : " + model);
```

Comment:
Možná by stálo za to dát zmínku do javadocu, že v případě nenalezení nevrací `null`, ale vyhazuje vyjímku.